Add info on bromodomain benchmark

GHeinzelmann opened the pull request:
Hi. I have finished writing the inroduction and the free energy calculations on BRD4(1), the only thing left now is choosing the benchmark ligands and creating the Table. I thought it would be good to have a checkpoint now, to see if everything is going in the right direction. Best,

Germano

davidlmobley commented:
Thanks! I'm dealing with a grant proposal right now but will try to have a look as soon as I can. Or @mkgilson can look too...

GHeinzelmann commented:
No worries! I'll start looking into the ligands to include in the mean time! Best,

Germano

mkgilson commented:
Thanks Germano! I will try to look this weekend. I am straining to 
finish editing the D3R GC2 overview paper, which is already late!

Best,
Mike

On 6/29/2017 11:25 AM, GHeinzelmann wrote:
>
> Hi. I have finished writing the inroduction and the free energy 
> calculations on BRD4(1), the only thing left now is choosing the 
> benchmark ligands and creating the Table. I thought it would be good 
> to have a checkpoint now, to see if everything is going in the right 
> direction. Best,
>
> Germano
>
> ------------------------------------------------------------------------
>
>
>         You can view, comment on, or merge this pull request online at:
>
> https://github.com/MobleyLab/benchmarksets/pull/36
>
>
>         Commit Summary
>
>   * Add files via upload
>   * Add files via upload
>   * Add files via upload
>   * Add files via upload
>   * Add files via upload
>   * Add files via upload
>   * Delete benchmarkset-GH.pdf
>   * Delete benchmarkset-GH.tex
>   * Add files via upload
>   * Add files via upload
>
>
>         File Changes
>
>   * *M* paper/benchmarkset.bib
>     <https://github.com/MobleyLab/benchmarksets/pull/36/files#diff-0>
>     (8858)
>   * *A* paper/benchmarkset[WIP].pdf
>     <https://github.com/MobleyLab/benchmarksets/pull/36/files#diff-1> (0)
>   * *A* paper/benchmarkset[WIP].tex
>     <https://github.com/MobleyLab/benchmarksets/pull/36/files#diff-2>
>     (1052)
>   * *A* paper/figures/brd4.pdf
>     <https://github.com/MobleyLab/benchmarksets/pull/36/files#diff-3> (0)
>
>
>         Patch Links:
>
>   * https://github.com/MobleyLab/benchmarksets/pull/36.patch
>   * https://github.com/MobleyLab/benchmarksets/pull/36.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/MobleyLab/benchmarksets/pull/36>, or mute the 
> thread 
> <https://github.com/notifications/unsubscribe-auth/AQEJQG0LSP2drvVDrrUU1Yg-36Cvk98yks5sI-wUgaJpZM4OJsWM>.
>

-- 
Michael K. Gilson, M.D., Ph.D.
Professor, Skaggs School of Pharmacy and Pharmaceutical Sciences
UC San Diego
9500 Gilman Drive
La Jolla, CA 92093-0736
voice: 858-822-0622
http://gilson.ucsd.edu
http://bindingdb.org
http://drugdiscovery.ucsd.edu
http://drugdesigndata.org

GHeinzelmann commented:
OK, just a small correction, sorry about that. It turns out some of the ligands of BRD4(1) are not neutral, so in the part that says that "all of the ligands are neutral" can be changed to "most ligands are neutral", or maybe something similar. Best,

Germano

davidlmobley commented:
@GHeinzelmann - I am starting to look at this. I took the liberty of:
a) Changing the title of the PR to reflect what this is doing
b) moving your "benchmarksets[WIP].pdf" and ".tex" to "benchmarksets.pdf" and "benchmarksets.tex"; be sure to "git pull origin GHeinzelmann" to pull down these changes

The reason (b) is appropriate is that you aren't actually changing these files until I agree to it, you're just PROPOSING changing them. So it's totally fine to make that proposal. AND, changing the ACTUAL FILES will make it so GitHub will show me what you're changing in the files (see "files changed" tab above!) in a useful way, rather than showing me that you've introduced two new files which consist of entirely new content.

davidlmobley commented on paper/benchmarkset.tex:
```diff
@@ -745,6 +745,102 @@ \subsubsection{Computational challenges posed by the T4 lysozyme benchmarks}
+We extend our protein benchmarks to protein domains called bromodomains, which have recently been the topic of numerous studies in the search for high affinity ligands, using both 
+computational~\cite{Deepak:2017:Chem.Biol.Drug.Des., Dickson:2016:J.Chem.Phys., Raj:2016:J.Biomol.Struct.Dyn., Ran:2015:Mol.Biosyst., Zhao:2014:Bioorg.Med.Chem.Lett, Xu:2016:J.Med.Chem., Yang:2016:BioorganicChemistry} and experimental 
+structure-based techniques~\cite{ Zhang:2013:J.Med.Chem.,Bamborough:2015:J.Med.Chem., Ran:2015:J.Med.Chem., Gosmini:2014:J.Med.Chem., Chung:2011:J.Med.Chem., Xue:2016:J.Med.Chem., Fill:2012:Bioorg.Med.Chem., Fish:2012:J.Med.Chem.}. 
+In living cells, bromodomains are responsible for recognizing and binding to acetylated lysine residues present in the histone tails, a known epigenetic marker~\cite{Chung:2016:Prog.Med.Chem.}. They also bind a broad spectrum of small to 
```
Small LaTeX-on-GitHub tip: Break lines at least every sentence (after every period), if not more often. That way, when we go in and make edits, it will be easy to see what was changed.

GHeinzelmann replied:
OK, I'll remeber to do that next time, sorry about that.

davidlmobley replied:
It actually doesn't make much difference this time because the material is all new, but for smaller changes it becomes more important.

davidlmobley commented on paper/benchmarkset.tex:
```diff
@@ -745,6 +745,102 @@ \subsubsection{Computational challenges posed by the T4 lysozyme benchmarks}
+fluctuations, all cases show a sharp irreversible transition on this loop generally after 20 ns of simulations, suggesting an initial metastable state followed by stabilization of the protein backbone. 
+
+Heinzelmann et al.~\cite{Heinzelmann:2017:J.Chem.TheoryComput.} have investigated this phenomenon in more detail, with simulations and free energy calculations showing that apo crystal structures of BRD4(1) are indeed metastable, 
+and after around 50 ns of simulations a transition is seen from the so-called closed to the open state (Fig. 3B). According to the free energy calculations performed using the Amberff14SB protein force field and the TIP3P water model, 
```
It seems somewhat odd here that it says 50 ns but a couple lines prior it says 20 ns. Which is it? Or is it "20-50 ns depending on simulation details", or...?

(One thing to try to do here: Be definitive about things which you KNOW, be conservative about things you THINK, and be clear in identifying things that are undecided/need exploring/might be important. The target audience is a non-expert in the system who needs to know what the important issues are, what's already known about them, and what they might (or might not!) need to explore...)

GHeinzelmann replied:
In my simulations always after 50ns, but in other studies it seemes to have happened before, particularly on Ref. 195 (SI) which uses the CHARMM force field. This reference shows a clear transition in the loop and the RMSD values match mine, but there is no way I can know how for sure if it is exactly the same thing.

davidlmobley replied:
OK. So maybe we can make that more clear so it doesn't sound like you're contradicting yourself.

davidlmobley commented on paper/benchmarkset.tex:
```diff
@@ -745,6 +745,102 @@ \subsubsection{Computational challenges posed by the T4 lysozyme benchmarks}
+contacts involving loop residues and neighbouring molecules in the crystal arrangement. This last possibility has been proposed before, when noting the inconsistency between the relatively uniform crystallographic B-factors and the high 
+flexibility of the ZA-loop observed in simulations of the apo CREBBP bromodomain~\cite{Spilio:2014:Isr.J.Chem.}. 
+
+The free energy of the apo BRD4(1) closed-to-open conformational change is independent of the substrate~\cite{Heinzelmann:2017:J.Chem.TheoryComput.}, so it will not impact the ligands $relative$ binding free energies, as long
```
Should this be "appears to be independent" or maybe indicate how this is known? Would you expect this to be true for all substrates...?

GHeinzelmann replied:
The way I have calculated this transition, it starts out with the mestastable closed state without any ligands, and then goes to the open state also with no ligands. So if I go straight from the metastable apo state (held only by the wall), to the apo open state, the ligand is never there so this part is independent of the ligand.

davidlmobley commented on paper/benchmarkset.tex:
```diff
@@ -745,6 +745,102 @@ \subsubsection{Computational challenges posed by the T4 lysozyme benchmarks}
+
+Many studies have focused on the calculation of binding free energies of small molecules to BRD4(1), using a variety of methods such as alchemical transformations~\cite{aldeghi_accurate_2016, Aldeghi:2017:J.Am.Chem.Soc.}, 
+attach-pull-release (APR)~\cite{Heinzelmann:2017:J.Chem.TheoryComput.}, adaptive biasing potentials (ABP)\cite{Dickson:2016:J.Chem.Phys.}, steered molecular dynamics (SMD)~\cite{ Muvva:2014:Mol.Biosyst.}, as well as molecular 
+mechanics combined with either Poisson-Boltzmann (MM-PBSA)~\cite{Su:2017:J.Biomol.Struct.Dyn.,Deepak:2017:Chem.Biol.Drug.Des.} or generalized Born (MM-GBSA) surface area~\cite{Raj:2016:J.Biomol.Struct.Dyn.}. The first two methods have 
```
We focus on "rigorous" free energy techniques here so we don't need to cite MM-PBSA/MM-GBSA work unless conclusions from it are relevant to this study. Are they, @GHeinzelmann ?

GHeinzelmann replied:
Not really, just thought it would be interesting to mention it, but nothing really relevant there,

GHeinzelmann replied:
Thank you as well! I would be happy to keep helping, with the corrections for this part and also any other contributions to the current benchmarks! :)

davidlmobley commented:
@GHeinzelmann - it looks like things are headed in the right direction, though I still need to have a look at whether organization of this is consistent with the other benchmarks. But on the whole the content seems solid and worthwhile, which I very much appreciate!!

I left some comments for you to look at when you have time, and potentially incorporate in an update to this PR.

I will probably also go through the parts you have written and make some edits to grammar/style and submit a PR to from my branch to your branch (so that you can review/merge it and then it will update this PR). But that probably won't be for a day or two, so in the meantime, please proceed!

One other thought -- if you're up for it, once you have this written up, it would probably be good to ask Matteo Aldeghi or Phil Biggin to have a look over it as sort of an informal peer review.

**Oh, finally -- your PR should also add yourself as an author!**

Thanks again for your contribution.

davidlmobley commented:
@GHeinzelmann - how does this stand? Do you think it's ready to merge? I'll have some time in transit this week (part of it is today) and hope to review your changes to finalize, if you think it's done.

GHeinzelmann commented:
Hi. I think pending a final review it is ready, After I got the openeye license I have redone the ligand table figures and also got the smiles for each molecule. I just could not find the PubChem ID for two of them, but the rest is all there. I was waiting for some feedback on the ligands before I uploaded the structures, but since we are merging soon I'll do that part now so everything is ready.

GHeinzelmann commented:
BTW is is on the GHeinzelmann branch.

GHeinzelmann commented:
BTW the last version is on the GHeinzelmann branch.

davidlmobley commented:
I was happy with the ligand choices.

davidlmobley commented:
@GHeinzelmann - I spotted one more issue. I can't parse the SMILES string for Cc1c(sc-2c1C(=N[C@H(c3c2c(no3)C)CC(=O)N)c4ccc(cc4)Cl)C , "Compound 3". Can you re-check please? It doesn't parse with my tools.

I did find a PubChem compound ID for RVX-OH, the other one where you were missing an ID, so this is the last one remaining.

GHeinzelmann commented:
OK, thanks for finding the PubChem ID for that compound, sorry I could not find it myself but I did try :). About the smiles for compound 3, I just rechecked and it it renders the 2D structure correctly, so not sure what could be the problem. I am pasting it again directly from the input file of the openeye 2D rendering:

Cc1c(sc-2c1C(=N\[C@H\](c3c2c(no3)C)CC(=O)N)c4ccc(cc4)Cl)C

It looks the same as the one you wrote, except for the [C@H part, which seems to be missing a closure bracket in yours. I just checked the paper table and it is written the same way as above, so that might fix it.

GHeinzelmann commented:
Aside from that, I am creating all the ligand and solvated structures to include in the input_files and that part will be finished today.

davidlmobley commented:
Ah, you seem to have nailed it, @GHeinzelmann . The string I'm seeing in the LaTeX file AND on GitHub is  `Cc1c(sc-2c1C(=N[C@H(c3c2c(no3)C)CC(=O)N)c4ccc(cc4)Cl)C` which is missing the close square bracket after the `C@H`, but the string you just pasted, `Cc1c(sc-2c1C(=N[C@H](c3c2c(no3)C)CC(=O)N)c4ccc(cc4)Cl)C`, as that and parses properly. Can you fix? 

And, yeah, for some reason I can't find this one in PubChem, which is odd... So, we have to leave N/A for that one for now.

GHeinzelmann commented:
Yes I'll fix that no problem. The .tex version I have here has the bracket, so it is possible that I accidentally erased it when I had the GitHub window open. Sorry about that.

GHeinzelmann commented:
I think we are all set and ready to merge. I figured I would add myself as a second author, not sure if that is the idea but it can be easily changed if needed. The input files and the README.md file are in the input_files/BRD4 folder, following Niel's organization in his branch. I have also addressed the review comments in the new .tex and .pdf versions of the main paper. I think I have covered everything that was missing, but let me know if there is anything else to be done. I think it all looks good, thanks for inviting me to contribute to the paper!

davidlmobley commented:
@GHeinzelmann - great, thanks! I'll take a final look. I just need to hear back from Mike...

GHeinzelmann commented:
Hi. I have updated the paper in my branch with the latest comments from @mkgilson, also changing a little the bit the information on Compound 4 and Compound 8a in Table 7 and in the section where we discuss this Table (last paragraph of subsection 1). For now I have kept these two ligands in the benchmarks, discussing the docking of Compound 8a (and the similarity to the pose of Compound 4) in the README from the input_files/BRD4 folder.

davidlmobley commented:
I'm running behind on the e-mail thread/updates here but hope to review this and get everything merged by the end of the week. Thanks for all your work on this!

davidlmobley commented:
@GHeinzelmann - congratulations, it is merged! You're now an author. :) Thanks for all your work on this.

GHeinzelmann commented:
That's great! Thanks for the reviews and the opportunity!

mkgilson commented:
:-)

On 8/4/2017 1:12 PM, David L. Mobley wrote:
>
> @GHeinzelmann <https://github.com/gheinzelmann> - congratulations, it 
> is merged! You're now an author. :) Thanks for all your work on this.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/MobleyLab/benchmarksets/pull/36#issuecomment-320342669>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AQEJQDOdqjbXIgOBXdDlTB9sACU_QaHzks5sU3s1gaJpZM4OJsWM>.
>

-- 
Michael K. Gilson, M.D., Ph.D.
Professor, Skaggs School of Pharmacy and Pharmaceutical Sciences
UC San Diego
9500 Gilman Drive
La Jolla, CA 92093-0736
voice: 858-822-0622
http://gilson.ucsd.edu
http://bindingdb.org
http://drugdiscovery.ucsd.edu
http://drugdesigndata.org
